src/mono/browser/runtime/invoke-js.ts

```diff
@@ -274,6 +279,7 @@ function bind_fn(closure: BindingClosure) {
     const fqn = closure.fqn;
     if (!WasmEnableThreads) (<any>closure) = null;
     return function bound_fn(args: JSMarshalerArguments) {
+        const is_async = WasmEnableThreads && is_receiver_should_free(args);
```

Comment:
nit: The `async` is strongly connected to tasks/promises, but it has a different meaning, right?

Reply:
I will rename it on the next PR